content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,236 @@
+* **D﻿uration:** All workshops are time bombed. You will define here the time allocated to perform the workshop
+* **A﻿ctive:** Tag to set to enable visibility of the workshop's tile in the registration portal
+* **W﻿orkshopImg:** As part of the lifecycle of the workshop, several emails are sent to the student. A workshop image is embedded in the first emails
+* **session type:** Workshops-on-Demand by default (automatically created at the import of the wod.yml file process)
+
+The following fields are required by the infrastructure. And I will work as the infrastructure Admin with Matt to define them.
+
+* **capacity:** The number of maximum concurrent students allowed to take on the workshop
+* **range:** The range between which students get picked at registration time
+* **r﻿eset and ldap** entries are to be used by backend server automation if dedicated reset scripts and ldap authentication are required by the workshop
+* **location:** If your setup includes multiple jupyterhub servers, use this field to allocate workshops according to your needs.
+* **compile:** This entry will be filled with the name of a script to be compiled at deployment time. This feature allows the admin to hide login scripts and credentials in non-editable executable files
+* **varpass:**  This defines whether or not a workshop requires a password variable needs to be leveraged
+* **B﻿adgeImg:** As part of the lifecycle of the workshop, several emails are sent to the student. In the final email, a badge is included. It allows the student to share its accomplishment on social media like linkedin for instance
+* **beta:** Not implemented yet :-)
+* **AlternateLocation:** future development. The purpose is to allow automation of the relocation of a workshop in case of primary location's failure
+* **ReplayLink:** YouTube link of the recorded video to be used as a replay
+* **Replayid:** This ID is used to link the correct video with the associated workshop. This is the replayId present in the workshops table (automatically created at the import of the wod.yml file process)
+* **MonoAppliance:** Some workshops require a single dedicated appliance
+* **MultiAppliance:** Some workshops require multiple dedicated appliances
+
+***N﻿ote:*** B﻿oth W﻿orkshopImg and B﻿adgeImg are delivered by the frontend web server. 
+
+I﻿f you feel you need more details about the registration process, please take a look at the **Register Phase** paragraph in [the following introductionary blog](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/).
+
+As Matt will create a simple workshop that does not require any infrastructure but the JupyterHub itself. As far as the infrastructure's requirements, a new kernel was needed. No additional scripts were required for this workshop.
+
+A﻿s an admin of the Workshops-on-Demand infrastructure, I had to perform several tasks on a development environment and a staging environment:
```

Comment:
Change "had" to "have". The way you have been portraying this example is either as something that will/can happen in the future or is happening now. So you need to stay in the same tense.